Source/pfile.cpp

```diff
@@ -164,7 +164,7 @@ static HANDLE pfile_open_save_archive(DWORD save_num)
 {
 	HANDLE archive;
 
-	if (SFileOpenArchive(GetSavePath(save_num).c_str(), 0x7000, FS_PC, &archive))
+	if (SFileOpenArchive(GetSavePath(save_num).c_str(), 0, 0, &archive))

```

Comment:
Doesn't removing 0x7000 change behaviour?

Here StormLib uses this flag:

https://github.com/ladislav-zezula/StormLib/blob/dc867243c099f0350229335f8e283cce2016105a/src/SFileOpenArchive.cpp#L216

Reply:
0x7000 went to dwPriority, either this isn't implemented in storm lib or we made a mistake by implementing this our selfs in `SFileOpenFile`